code/drasil-system/package.yaml

```diff
@@ -18,6 +18,7 @@ dependencies:
 - pretty
 - lens
 - containers
+- drasil-data
```

Comment:
I'm not sure how I feel about `drasil-system` being dependant on `drasil-data`, but it might mean that `drasil-data` could be split up.

Reply:
Moved to `drasil-metadata` for the "new" concepts we need. I'll move the rest in a subsequent PR unless you prefer I do that in this one as well.